README.md

```diff
@@ -618,6 +619,29 @@ chatSession.onTyping(event => {
 Subscribes an event handler that triggers whenever a `application/vnd.amazonaws.connect.event.typing` event is created by any participant.
 The `data` field has the same schema as `chatSession.onMessage()`.
 
+ ##### `chatSession.onParticipantInvited()`
```

Comment:
is this feature already launched?

Reply:
Feature emitting this event is launching on 12/18.